files/gen_chkrootkit_expected_output.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+LOG_DIR=/var/log/chkrootkit
+
+eval /usr/sbin/chkrootkit -q > $LOG_DIR/log.today.raw 2>&1
+# the sed expression replaces the messages about /sbin/dhclient3 /usr/sbin/dhcpd3
+# with a message that is the same whatever order eth0 and eth1 were scanned
+sed -r -e 's,eth(0|1)(:[0-9])?: PACKET SNIFFER\((/sbin/dhclient3|/usr/sbin/dhcpd3)\[[0-9]+\]\),eth\[0|1\]: PACKET SNIFFER\([dhclient3|dhcpd3]{PID}\),' \
```

Comment:
Maybe we could have our own cron script? We could put it in place during deployment.
But I don't know if  it would be a good idea to deviate from the distro.

What do you think?